### ISSUE TITLE
Deprecation of `sdimaging`

### ISSUE THREAD

**Issue by tnakazato** [casangi/casadocs#111]
Since we decided to deprecate `sdimaging` as of CASA 6.7.0, Relevant casadocs pages must be updated (or removed) as needed.

**Comment by bemonts** [comment]
Hi @tnakazato , looking at CAS-14439, what we probably need to do is add a deprecation warning to the sdimaging task page, and maybe change references from sdimaging to tsdimaging throughout the docs. Then in a future CASA version sdimaging (and all remaining references) to it can be removed from the docs. Is that correct? Please let me know and I can update this directly on CASA Docs.

**Comment by tnakazato** [comment]
@bemonts, that is correct. Thank you for your offer! Please update CASA Docs and let me know once you have done.

**Comment by tnakazato** [comment]
The sdimaging task has been removed as of CAS 6.7.3. Documentation has been updated accordingly. I'll close this ticket.
